main.py

```diff
@@ -40,5 +59,9 @@
                 print(e)
             end = timer()
             print(end - start, "seconds")
+            total += (end-start)
+    print(f"Inserted {len(json_files)} documents in {total} seconds")
+    # srch = store.search(resource='patient',params="male")
```

Comment:
Should be remove these two lines of comment?

Reply:
Sure!